README.md

```diff
@@ -141,10 +149,105 @@ Use #AARRGGBB for colors instead of ##AABBGGRR, to be compatible with Flutter im
 
 #### Image foreground is too big / too small
 
-For best results try and use a foreground image which has padding much like [the one in the example](https://github.com/fluttercommunity/flutter_launcher_icons/blob/master/example/assets/images/icon-foreground-432x432.png).
+For best results try and use a foreground image which has padding much like [the one in the example](example/default_example/assets/images/icon-foreground-432x432.png).
 
 [Related issue](https://github.com/fluttercommunity/flutter_launcher_icons/issues/96)
 
+#### Can't run web example project!
+
+Make sure that you have enabled web support!
+
+To check whether you have, try running `flutter devices`.
+
+You should see a line similar to the following in its output:
+
+```
+Web Server (web) • web-server • web-javascript • Flutter Tools
+```
+
+If you don't, please [enable web support!](https://flutter.dev/docs/get-started/web#set-up)
+
+#### Launcher icons not present in debug mode
+
+`favicon.png` may not be served with `index.html` when `flutter run -d web-server` is used. Try this!
+```bash
+$ flutter build web
+$ cd build/web
+$ python3 -m http.server
+```
+
+Favicons should now be present!
+
+#### `flutter run --flavor` doesn't work with web!
+
+`flutter run --flavor` is currently an [`android` and `iOS` specific feature!](https://github.com/flutter/flutter/issues/59388)
+
+To make this work with `flutter_launcher_icons`, say that you have the following flavor-specific configuration files:
+
+`flutter_launcher_icons-development.yaml`
+```yaml
+flutter_icons:
+  android: true
+  ios: true
+  image_path: "assets/launcher_icon/demo-icon-dev.png"
+```
+
+`flutter_launcher_icons-integration.yaml`
+```yaml
+flutter_icons:
+  android: true
+  ios: true
+  image_path: "assets/launcher_icon/demo-icon-int.png"
+```
+
+`flutter_launcher_icons-production.yaml`
+```yaml
+flutter_icons:
+  android: true
+  ios: true
+  image_path: "assets/launcher_icon/demo-icon.png"
+```
+
+We can tell `flutter_launcher_icons` to only generate web icons for the `production` flavor:
+
+`flutter_launcher_icons-development.yaml`
+```yaml
+flutter_icons:
+  android: true
+  ios: true
+  web: false
+  image_path: "assets/launcher_icon/demo-icon-dev.png"
+```
+
+`flutter_launcher_icons-integration.yaml`
+```yaml
+flutter_icons:
+  android: true
+  ios: true
+  web: false
+  image_path: "assets/launcher_icon/demo-icon-int.png"
+```
+
+`flutter_launcher_icons-production.yaml`
+```yaml
+flutter_icons:
+  android: true
+  ios: true
+  web: true
+  image_path: "assets/launcher_icon/demo-icon.png"
+```
+
+Using this, `flutter run --flavor <some flavor here>` will work for iOS and Android
+and `web` will always use the production icon!
+
+We make this change here, rather than in `pubspec.yaml` because
+`flutter_launcher_icons` ignores any configuration information in
+`pubspec.yaml` when flavor config files are present!
+
+For a working example, please see [`example/flavors`](example/flavors)!
```

Comment:
If `flavors_renameme` is renamed to something other than `flavors`, this link will need to be updated!